test/api/v3/integration/user/auth/POST-login-local.test.js

```diff
@@ -43,7 +45,7 @@ describe('POST /user/auth/local/login', () => {
     })).to.eventually.be.rejected.and.eql({
       code: 401,
       error: 'NotAuthorized',
-      message: t('accountSuspended', { userId: user._id }),
+      message: t('accountSuspended', { communityManagerEmail: nconf.get('EMAILS:COMMUNITY_MANAGER_EMAIL'), userId: user._id }),
```

Comment:
sorry, I meant to post this comment in the two auth.js files, doing it in the tests is not necessary indeed